packages/web/components/elements/images/OmnivoreNameLogo.tsx

```diff
@@ -29,12 +29,7 @@ export function OmnivoreLogoIcon(props: OmnivoreLogoProps): JSX.Element {
       fill="none"
       xmlns="http://www.w3.org/2000/svg"
     >
-      <path
-        d="M9.13628 18.637V10.8781C9.13628 10.2147 9.91506 9.8397 10.4342 10.3012L12.8283 13.7913C13.3186 14.1951 14.0108 14.1951 14.5012 13.7913L16.8375 10.33C17.3567 9.89738 18.1355 10.2435 18.1355 10.9069V15.0027C18.1355 16.9929 19.4623 18.6081 21.4525 18.6081H21.5101C23.2119 18.6081 24.6829 17.4544 25.0867 15.8103C25.2886 14.945 25.4617 14.0508 25.4617 13.3586C25.4329 6.58038 19.693 1.44624 12.8283 1.90774C7.00186 2.31154 2.30037 7.01303 1.89657 12.8394C1.43507 19.7042 6.85765 25.444 13.6647 25.444"
-        stroke={props.strokeColor}
-        strokeWidth="2.29961"
-        strokeMiterlimit="10"
-      />
+    <path d="M7.11147 14.2474V8.41955C7.11147 7.92126 7.69642 7.63962 8.08639 7.98626L9.88456 10.6077C10.2529 10.911 10.7728 10.911 11.1411 10.6077L12.896 8.00792C13.2859 7.68295 13.8709 7.94293 13.8709 8.44122V11.5176C13.8709 13.0125 14.8675 14.2257 16.3623 14.2257H16.4057C17.6839 14.2257 18.7888 13.3591 19.0921 12.1242C19.2437 11.4743 19.3737 10.8027 19.3737 10.2827C19.3521 5.1915 15.0408 1.33517 9.88456 1.68181C5.50828 1.98512 1.97692 5.51647 1.67361 9.89276C1.32698 15.049 5.39996 19.3603 10.5128 19.3603" stroke={props.strokeColor} strokeWidth="1.72727" strokeMiterlimit="10"/>
```

Comment:
I don't think we should have any logo changes in here, since this is used in the primary header.

Reply:
We have reverted these changes. Had to increase the icon size from 24px to 26px otherwise it would look cropped out. Changed all social icon sizes to 26px as well so they match. WDYT?